apiserver/facades/client/client/status.go

```diff
@@ -247,6 +259,14 @@ func (c *Client) FullStatus(args params.StatusParams) (params.FullStatus, error)
 					matchedSvcs.Add(unit.ApplicationName())
 					continue
 				}
+				if machineId != "" {
+					_, ok := machineUnits[machineId]
```

Comment:
You don't need this ok check.

The else clause here works fine for the initial case too as a typed nil is returned if not found.

Reply:
yes, looks so much leaner \o/ Thank you!